game/static/game/css/board.css

```
@@ -2034,6 +2034,128 @@ body {
+    backdrop-filter: blur(10px);
+    -webkit-backdrop-filter: blur(10px);
+    z-index: 99999;
+    padding: 24px;
+}
+
+.shortcuts-modal.active {
+    display: flex;
+    animation: shortcutsModalFadeIn 0.3s ease-out forwards;
+}
+
+@keyframes shortcutsModalFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "shortcutsModalFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)